packages/web-twig/src/Twig/PropsExtension.php

```diff
@@ -112,12 +124,48 @@ public function renderStyleProp(Environment $environment, array $props): string
         ]);
     }
 
+    /**
+     * Converts a mixed value to a string.
+     *
+     * @param mixed $value The value to convert.
+     * @return string The value as a string.
+     */
+    private function toString($value): string
```

Comment:
How much "mixed" can the value be? 🙂 Do we need to handle all these cases for `PARAMETER_SPIRIT_CSS_CLASS_PREFIX`?